readme.md

```diff
@@ -50,6 +50,7 @@ If you're interested in documenting or verifying the behaviour of the original M
 * Partially supports some features of Minecraft Classic versions before 0.30
 * Lightweight, minimal memory usage compared to original Minecraft Classic
 * Much better performance than original Minecraft Classic
+* It does **not** support Java (or require one) or UWP libraries.

```

Comment:
Obviously a game doesn't support a programming language, along with libraries for literally anything that couldn't be used for a plugin.